src/app/_components/_sharedcomponents/Cards/GameCard/GameCard.tsx

```diff
@@ -31,22 +47,35 @@ const GameCard: React.FC<IGameCardProps> = ({
 	const styles = {
 		cardStyles: {
 			borderRadius: ".38em",
-			height: size === "standard" ? "10rem" : "8rem",
-			width: size === "standard" ? "7.18rem" : "8rem",
+			...(variant === "lobby"
+					? {
+						height: "18vh",
```

Comment:
In general we want to avoid using view sizes (vh, vw) for elements that aren't related to the page layout. We want the card size to be predicatble given the container that it's in and basing that size off the whole window makes that more difficult. Lets just make this a good default size using rem and then we can take a closer look at sizing when we do a responsive pass later.